src/sentry/tasks/post_process.py

```diff
@@ -480,6 +480,29 @@ def should_update_escalating_metrics(event: Event) -> bool:
     )
 
 
+@instrumented_task(
+    name="sentry.issues.tasks.post_process.post_process_group",
+    time_limit=120,
+    soft_time_limit=110,
+    silo_mode=SiloMode.REGION,
+    taskworker_config=TaskworkerConfig(
+        namespace=ingest_errors_postprocess_tasks,
+        processing_deadline_duration=120,
+    ),
+)
+def post_process_group_shim(*args, **kwargs) -> None:
```

Comment:
Long term this function will go away and the function will continue to be named `post_process_group`.